gradle/libs.versions.toml

```diff
@@ -87,8 +86,9 @@ springboot-core = { module = 'org.springframework.boot:spring-boot', version.ref
 springboot-starter-logging = { module = 'org.springframework.boot:spring-boot-starter-logging', version.ref = 'springboot' }
 springboot-starter-tomcat = { module = 'org.springframework.boot:spring-boot-starter-tomcat', version.ref = 'springboot' }
 tomcat-jdbc = { module = 'org.apache.tomcat:tomcat-jdbc', version.ref = 'tomcat' }
+springboot-starter-test = { module = 'org.springframework.boot:spring-boot-starter-test', version.ref = 'springboot' }
```

Comment:
Can we keep these alphabetically ordered?

Reply:
Done.